CONTRIBUTING.md

```diff
@@ -46,7 +46,14 @@ make the build system to sign the module, if your kernel rejects unsigned ones).
 `sensors` command should show an entry named "asusec-..." with the sensor readings (`sensors 'asusec-*'` will show only
 that entry). Please note the blank value for temperature sensors: -40 (0xD8).
 
-Please ensure that all the list entries you add to the source files keep the alphabetical order of the lists.
+Please ensure that all the list entries you add to the source files keep the alphabetical order of the lists. These
+checks will return nothing if the relevant sections are in the right order:
```

Comment:
I like this! Could you, please, put that into a sctipt (e.g. `./scripts/check.sh`)?